shared/macros/10-bash.jinja

```diff
@@ -304,7 +304,7 @@ fi
         parameter=parameter,
         value=value,
         create=true,
-        insert_after="",
+        insert_after="[Coredump]",
```

Comment:
Is the problem that insert_after should be a regular expression, which means that this matches some class of characters instead of the literal string?

Reply:
Good catch, fixed.